apps/app/src/views/components/SplitLayout.tsx

```diff
@@ -10,14 +10,14 @@ export const SplitLayout = (props: any) => {
     <Stack
       flexDir={isMobileAndTablet ? 'column' : 'row'}
       gap={5}
-      width="95vw"
+      width="100%"
       maxW="1400"
       justify="center"
     >
       <Box
-        width={isMobileAndTablet ? '100vw' : 'full'}
+        width={isMobileAndTablet ? '100%' : 'full'}
         alignSelf={isMobileAndTablet ? 'center' : 'flex-start'}
-        pt={2}
+        mt={2}
         px={isMobileAndTablet ? 4 : 0}
```

Comment:
<img width="457" alt="Screenshot 2024-09-05 at 3 05 02 PM" src="https://github.com/user-attachments/assets/1d717823-76ab-47a4-88bd-09dee32cae21">

Think this would look even sweeter if we can remove the left and right padding on mobile/tablet 🙏